common/helpers/events/get-description.js

```diff
@@ -1,13 +1,38 @@
 const i18n = require('../../../config/i18n')
 
+const noSupplierDescriptions = [
+  'PerViolentDangerous',
+  'PerWeapons',
+  'PerConcealed',
+  'PerSelfHarm',
+  'PerEscape',
+  'PerMedicalAid',
+  'PerMedicalDrugsAlcohol',
+  'PerMedicalMentalHealth',
+  'PerMedicalMedication',
+  'PerPropertyChange',
+  'PerGeneric',
+  'PersonMoveDeathInCustody',
+  'PersonMoveUsedForce',
+]
+
 const getDescription = event => {
-  const { event_type: eventType, details } = event
+  const { event_type: eventType, details, supplier } = event
   details.vehicle_reg =
     details.vehicle_reg || details.journey?.vehicle?.registration
 
-  const description = i18n
-    .t(`events::${eventType}.description`, details)
-    .replace(/^(\s*<br>)+/, '')
+  let description
+
+  if (supplier === null && noSupplierDescriptions.includes(eventType)) {
+    description = i18n
+      .t(`events::${eventType}.no_supplier_description`, details)
+      .replace(/^(\s*<br>)+/, '')
+  } else {
+    description = i18n
+      .t(`events::${eventType}.description`, details)
+      .replace(/^(\s*<br>)+/, '')
+  }
```

Comment:
Rather than putting this logic here, we could use a feature of i18next called contexts: https://www.i18next.com/translation-function/context

So in the strings file we could have something like this where there needs to be a different translation:

```json
    "description": "{{move.supplier.name}} staff {{supplier_personnel_numbers}} used force on $t(events::person) <br>The incident was reported at {{reported_at, formatTime}} <br>$t(events::select_fault_classification)",
    "description_without_supplier": "{{move.supplier.name}} staff {{supplier_personnel_numbers}} used force on $t(events::person) <br>The incident was reported at {{reported_at, formatTime}} <br>{{notes}}"
```

And then I imagine we would use it something like this:

```js
if (supplier === null) {
  details['context'] = 'without_supplier'
}

const description = i18n
  .t(`events::${eventType}.description`, details)
  .replace(/^(\s*<br>)+/, '')
```

Then we wouldn't need the logic with `noSupplierDescriptions` as it would automatically use `events::${eventType}.description` unless the supplier is `null` and there's an `description_without_supplier` string available.

One big advantage of this is that for events where the description is the same, you wouldn't need to do this:

```json
    "description": "{{notes}}",
    "no_supplier_description": "{{notes}}"
```

Instead you'd just need:

```json
    "description": "{{notes}}",
```

And it would know that because there isn't a `_without_supplier` variation, it just uses the default variation.